makei/crtfrmstmf.py

```diff
@@ -118,11 +118,11 @@ def setupEnv(self):
             self.job.run_cl(f"CHGCURLIB CURLIB({self.env_settings['curlib']})", log=True)
 
         if "preUsrlibl" in self.env_settings and self.env_settings["preUsrlibl"]:
-            for libl in self.env_settings["preUsrlibl"].split():
+            for libl in self.env_settings["preUsrlibl"].split().reverse():
                 self.job.run_cl(f"ADDLIBLE LIB({libl}) POSITION(*FIRST)", log=True)
 
         if "postUsrlibl" in self.env_settings and self.env_settings["postUsrlibl"]:
-            for libl in self.env_settings["postUsrlibl"].split():
+            for libl in self.env_settings["postUsrlibl"].split().reverse():
```

Comment:
Do not revers the postUsrlibl inserts as they are added at the *LAST position and so will have their order preserved